### ISSUE TITLE
Whisper speech to text timeout

### ISSUE THREAD

**Issue by Ahmed-farag36** [home-assistant/intents#1361]
**My setup**
Virtual box running HA OS latest version 10.1 with dedicated resources: 4 cores CPU i7 100% cap and 8 GB of RAM and on SSD drive.
config for whisper is language: en, model: tiny-int8, beam-size: 1.
my pipeline is the default English with Whisper as SST and Google translate as TTS

**The problem**
sometimes I got my voice transcribed but in more than 25 seconds, but in many tries I get timeout error.

and this is my debug result
[conversation.zip](https://github.com/home-assistant/intents/files/11425802/conversation.zip)

any suggestions please

**Comment by edurenye** [comment]
I was having the same problem, so I tried to set the whisper docker image in base machine using docker in hopes to get better performance using the GPU (Virtual box does not have GPU passthrough, docker does support it). But the images do not have support for GPU, so I created this issue there: https://github.com/rhasspy/wyoming-addons/issues/3
If that is addressed we could use GPU and speed up Whisper.

Onother solution would be to increse the timeout, but I don't think it makes sence to have a timeout greater that 25 seconds, what we really need is a faster whisper server. IMHO.
